DANS-KNAW/easy-solr4files-index
# EASY-1320 custom ansible for file indexing

Fixes EASY-1230

#### When applied it will
* have a local VM in the project
* for a next pull request
  * do not duplicate the solr schema in dtap
  * fix the `attr_` prefixes for easy fields

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

in the project root
* `mvn clean install`
* `vagrant up`
  note that it takes a while to copy the bags 
* `curl http://test.dans.knaw.nl:8983/solr/fileitems/query\?q\=%2A%3A%2A\`
  it should find nothing
* `vagrant ssh`
* `easy-update-solr4files-index init`
  it should report it indexed two bags for the pdbs store and nothing for the other stores
* `exit`
* `curl http://test.dans.knaw.nl:8983/solr/fileitems/query\?q\=%2A%3A%2A\&fl=id\&rows\=1`
  it should find 36 files
* `vagrant provision`
  note that it does not copy the bags again (unless your removed `/data/bag-stores/pdbs`)

The dev-tool scripts still run with deasy.


#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-update-solr4files-index                      | [PR#4](https://github.com/DANS-KNAW/easy-update-solr4files-index/pull/4)     | predecessor
